docs/reference/ml/apis/delete-forecast.asciidoc

```diff
@@ -0,0 +1,64 @@
+[role="xpack"]
+[testenv="platinum"]
+[[ml-delete-forecast]]
+=== Delete Forecast API
+++++
+<titleabbrev>Delete Forecast</titleabbrev>
+++++
+
+Deletes forecasts from a {ml} job.  
+
+==== Request
+
+`DELETE _xpack/ml/anomaly_detectors/<job_id>/_forecast` +
+
+`DELETE _xpack/ml/anomaly_detectors/<job_id>/_forecast/<forecast_id>` +
+
+`DELETE _xpack/ml/anomaly_detectors/<job_id>/_forecast/_all` 
+
+
+==== Description
+
+By default, forecasts are retained for 14 days. You can specify a different 
+retention period with the `expires_in` parameter in the <<ml-forecast,forecast jobs API>>. The delete forecast API enables you to delete one or more forecasts before they expire.
+
+NOTE: When you delete a job its associated forecasts are deleted. 
+
+For more information, see {xpack-ref}/ml-overview.html#ml-forecasting[Forecasting the Future].
+
+
+==== Path Parameters
+
+`job_id` (required)::
+  (string) Identifier for the job.
+
+`forecast_id`::
+  (string) A comma-separated list of forecast identifiers. 
+//TBD: Can this ID be retrieved somehow?
```

Comment:
ID is only available from the initial creation of the forecast. 

This API is most likely be utilized from the UI (and they know how to gather and display forecasts from the `.ml-shared` index, or as an "oops, I created a forecast via API with the wrong settings".

Reply:
OK, so there are no immediate plans for a "GET forecasts" API?

Reply:
@lcawl no, there is not.